app/components/ui/WeGlot.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from "react";
+import { Helmet } from "react-helmet-async";
+
+const onLoad = (apiKey: string, targetLangCode: string | null) => {
+  // HACK: suppress TS compile errors when accessing the global WeGlot instance created by the script.
+  const weglot = (window as any).Weglot;
+  weglot.initialize({
+    api_key: apiKey,
+    auto_switch: false,
+    // Force WeGlot to avoid translating elements with this class
+    // TODO: we should find some method to set both ".weglot-no-translate" class AND translate="no" attribute
+    // whenever we have to specify not translating an html element.
+    excluded_blocks: [
+      {
+        value: ".weglot-no-translate",
+      },
+    ],
```

Comment:
Offhand, I don't know of any functional difference between `.notranslate` CSS classes and the `translate="no"` attr. I went with translate="no" initially as a kind of separation of concerns between style and translation. But if ` .notranslate` works for Google and WeGlot, I'd be in full favor of using that.

Reply:
Cool, I'll try it out this weekend and report back if `.notranslate` works.
As for `excluded_blocks`... I'll try that out too, but I have less confidence in WeGlot support for that one.

Reply:
Here are the results of my tests:
- `.notranslate` does work and is recognized by the Google Translate widget. Hurray!
- Neither `translate="no"` nor `[translate="no"]` seemed to work as WeGlot `excluded_blocks`. The latter made the entire page silently fail to translate.

So given this, my next move will be to push a commit that uses `.notranslate`. What do we want to do - keep both `translate="no"` and `.notranslate`, and then remove `translate="no"` in a separate PR? or just replace all instances of `translate="no"` with `.notranslate` in this one?

Reply:
Awesome, thanks for testing it out! I agree with @schroerbrian that removing the `translate="no"` in this PR would be great, but it can also come in a followup PR if that's easier.